Create the rails project

DiegoSteve opened the pull request:
# Description
What did you implemented/Why did you implemented this:
 -What?: Create a initialize rails project
-Why?: Was created to set all the initial configuration to work with Rails

 # Testing
 Help me how can I test or look at the changes

  ## Screenshots
![Captura de Pantalla 2022-03-08 a la(s) 17 55 04](https://user-images.githubusercontent.com/72957263/157345777-2be945cf-a4d8-4c95-9ec7-b5a4bc57e34c.png)

DiegoSteve committed:
Create the rails project

Co-authored-by: Frcomparan <frcomparan@gmail.com>
Co-authored-by: brocha05 <brocha_05@hotmail.com>
Co-authored-by: HeikelV <heikelvillar@gmail.com>

JuanequeX commented on config/database.yml:
```diff
@@ -0,0 +1,86 @@
+  #schema_search_path: myapp,sharedapp,public
+
+  # Minimum log levels, in increasing order:
+  #   debug5, debug4, debug3, debug2, debug1,
+  #   log, notice, warning, error, fatal, and panic
+  # Defaults to warning.
+  #min_messages: notice
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: kata_07_reloj_checador_k7_f22_t4_test
+
+# As with config/credentials.yml, you never want to store sensitive information,
+# like your database password, in your source code. If your source code is
+# ever seen by anyone, they now have access to your database.
+#
+# Instead, provide the password or a full connection URL as an environment
+# variable when you boot the app. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# If the connection URL is provided in the special DATABASE_URL environment
+# variable, Rails will automatically merge its configuration values on top of
+# the values provided in this file. Alternatively, you can specify a connection
+# URL environment variable explicitly:
+#
+#   production:
+#     url: <%= ENV["MY_APP_DATABASE_URL"] %>
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
+production:
+  <<: *default
+  database: kata_07_reloj_checador_k7_f22_t4_production
+  username: kata_07_reloj_checador_k7_f22_t4
+  password: <%= ENV["KATA_07_RELOJ_CHECADOR_K7_F22_T4_DATABASE_PASSWORD"] %>
```
This file has your project's env-vars and should be in your .gitignore file
should be replaced with the file database.example.yml to have references to which username and database to use when someone downloads your project.

YilKanda replied:
I agree with @JuanequeX

Frcomparan replied:
Thanks for the review, we did the suggested changes

YilKanda reviewed (changes requested):
Remember, you must leave the database.yml.example file so that the user who downloads your project knows that there must be a file called database.yml

brocha05 commented:
We already add a file named database.example.yml in config directory, there we add an example of the neccesary values for the correct function of the database.

Frcomparan committed:
Add instructions about the function of files
Adding instructions for file <database.example.yml>
Adding instructions for file <.env.example>

Co-authored-by: brocha05 <brocha_05@hotmail.com>
Co-authored-by: HeikelV <heikelvillar@gmail.com>
Co-authored-by: DiegoSteve <diegosteve123321@gmail.com>

HeikelV reviewed (approved):
LGTM

brocha05 reviewed (approved):
(Y)

JuanequeX reviewed (approved):
LGTM
